### PR TITLE
Set up PHP in 1st CI job

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

**Comment by ksassnowski** [comment]
Can you provide a bit of context about how this change helps? I assume it's because the setup step could be cached this way?

**Comment by szepeviktor** [comment]
There are actual technical things but more important is the hierarchical order:

- CPU architecture
- OS
- OS packages
- language (e.g. PHP + PHP extensions)
- language packages (e.g. Composer packages)
- doing things with project source at the very end

PHP was in the **middle**, so I moved it up.

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -29,17 +36,10 @@ jobs:
...
-        uses: shivammathur/setup-php@v2
-        with:
-          php-version: ${{ matrix.php }}
-          extensions: pdo, sqlite, pdo_sqlite
-          coverage: none
-
       - name: Install dependencies
         run: |
           composer require "laravel/framework:${{ matrix.laravel }}" "orchestra/testbench:${{ matrix.testbench }}" --no-interaction --no-update
-          composer update --${{ matrix.dependency-version }} --prefer-dist --no-interaction --no-suggest
```
Comment (ksassnowski) [inline .github/workflows/run-tests.yml:42]: Any particular reason why you removed this?

Comment (szepeviktor) [reply .github/workflows/run-tests.yml:42]: As that is deprecated in Composer v2
https://getcomposer.org/upgrade/UPGRADE-2.0.md

Comment (ksassnowski) [reply .github/workflows/run-tests.yml:42]: Ah, good catch! I didn't know that

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**